### DISCUSSION TITLE
Lightning Node Routing Management and its Hidden Knowledge, Wink, Wink...

### DISCUSSION THREAD

**Discussion by BonzoDogDooDah** [lightningnetwork/lnd#7940] [Q&A]
Hello to all you Lightningnetwork Github developers, moderators, and posters,

We have a lightning node and are searching for some educated advice on how to run a more useful routing node. We were wondering if any of you here have any tips on where one might go to locate such hidden knowledge.

A short explanation of what we have been experiencing.

We have a small node with about 60 million sats that currently has six channels. We recently closed a few channels that did not seem to be doing very much (over a long period of time) to assist our node's routing figures and soon after that we opened a few much larger sat channels with big Lightning nodes like LNBig, Nicehash, etc... but unfortunately things only got worse and not better? So we lowered the fees considerably in hopes of improving things but as of yet we have not had much luck with little or no routing coming in and then out again and a lot of difficulty rebalancing using the RTL app as we are only a year at this and have only run a few terminal commands in lightning shell over this time period? Which prior to closing these long held channels appeared to have worked significanlty better that at the present time?

We took advice from a few contacts we have made over the last year but we must have misinterpreted this information because it has not worked that well for us. One of the main things we would like to know is how to gain more than one or two nodes that appear to us to start the routing ball rolling and route sats in to our node , as that seems to be the key here if we are not sadly mistaken? If we do not have at least one node routing sats into our node then nothing seems to happen other than stagnation?

Any ideas on how to create a more useful routing node for all of us in thisLighning Community and how to better manage a routing node that you may wish to offer to us would be greatly appreciated.

Yours in Lightning & Bitcoin

Thank you.

Orange_Bonzo

**Comment by bhandras** [comment]
Hi! I recommend following https://twitter.com/alexbosworth on twitter as he often shares tips about routing. In general you want to ensure good inbound liquidity (source) for your node and route traffic where it is needed (sinks). It is of course more nuanced than this, but try to find good sinks like LOOP (https://amboss.space/node/021c97a90a411ff2b10dc2a8e32de2f29d2fa49d41bfbb52bd416e460db0747d0d) or any large wallets or exchanges who are always in need of good inbound. Try to think in volume, as in how much do I need to route to pay for my expenses (channel open, close, loop in, out etc). Size your channels accordingly. Once you've found your edge, try and do bulk channel opens to save on fees. Check out Lightning Terminal (https://github.com/lightninglabs/lightning-terminal) to help with all this and also try out autopilot which will automate your channel fees to ensure better flows. All the best!

**Comment by Liongrass** [comment]
Hi!
Congrats on getting started with your routing node! It's indeed a difficult but fun journey, and there isn't one strategy that works for all. Think of yourself as somebody who makes money from information, more specifically the knowledge of who is sending money to whom. In a network with the privacy properties of Lightning, this can be impossible to know, and it's constantly changing too. But as you peer with new nodes, you will learn about how they behave, and more importantly, how they interact with each other.

There are many different kinds of nodes out there, and they have different liquidity needs:
- Other routing nodes. If you peer with them, see if you can minimize the overlap you have with these nodes, e.g. how many common peers you have
- Merchant nodes. Merchant nodes primarily receive payments, but they may behave differently. Some merchants will continuously push sats back out of their node to maximize their inbound. You'll unlikely make money routing sats to them, but you may make money routing sats from them! Other merchant nodes let their sats accumulate in the channel, then occasionally use services like Loop Out to empty them. Again other nodes may not manage their nodes at all, or close the channel as soon as sats have flowed to their side. Fees need to be high enough, and channels big enough, for this to be worth it!
- User nodes. These nodes primarily send sats out, but they can be hard to find. Some will refill their channels with Loop In, others will peer with Loop or buy sats from exchanges that offer Lightning withdrawals.

Eventually you'll have to find your "edge." One recommendation I have is to use the Lightning Network as often as possible for payments yourself, and be observant about which service you are paying through which nodes, what the fee levels are and how many channels the have. You'll be surprised to learn that a lot of services only have a small handful of channels, so if you can find them, you will have found your niche and collect sats.

At the moment, a big portion of the Lightning Network evolves around people acquiring inbound capacity in creative ways, then selling it to Loop or Bitrefill. Try it out, allow yourself to lose some sats in the first few months experimenting with swaps, deposits, circular routes etc. You'll learn a lot about the network that way too.

Lastly, iterate. If a channel doesn't seem to route, close it and deploy your capital elsewhere. If a channel routes, keep it. By constantly cycling your "bad" channels, you'll have more and more "good" channels.

We do have [some guides here](https://docs.lightning.engineering/the-lightning-network/liquidity/understanding-liquidity) that will help you understand liquidity, inbound and channel management. You may also find some good chat groups round that topic, such as Plebnet. All the best on your journey!

**Comment by BonzoDogDooDah** [comment]
Hi Leonhard,

Thank you for all the useful information, I will put it to good use.

Regards,
Orange_Bonzo
-- 
 Sent with Tutanota, enjoy secure & ad-free emails. 



31 Aug 2023, 07:47 by ***@***.***:
